app/controllers/accounts_controller.rb

```diff
@@ -70,12 +73,20 @@ def set_account
   end
 
   def next_url
+    pagination_url(max_id: @statuses.last.id)
+  end
+
+  def prev_url
+    pagination_url(since_id: @statuses.first.id)
```

Comment:
The reason there's no previous button is because since_id doesn't do what a previous button should do. since_id will display the most recent items potentially cut off by the ID, not the items *immediately before* the ID.